.test-defs/CreateShootForMirgration.yaml

```diff
@@ -0,0 +1,42 @@
+kind: TestDefinition
+metadata:
+  name: create-shoot
```

Comment:
This name should be unique in one repo.
Whats the difference to https://github.com/gardener/gardener/blob/master/.test-defs/CreateShoot.yaml ?